schedule/qam/common/qam-clamav.yaml

```diff
@@ -0,0 +1,10 @@
+name: qam-clamav
+description:    >
+    Maintainer: QE Core.
+    Extract 'clamav' module from 'mau-extratests1' to meet the RAM
+    requirements.
+schedule:
+    - installation/bootloader_start
+    - boot/boot_to_desktop
+    - console/clamav
```

Comment:
@pablo-herranz can you add a comment in the clamav module about this?

Reply:
Comment added to clamav.pm

~~~ perl
# NOTE: As the vendor states, clamav needs at least 2GB of RAM to work smooth. 
# To avoid interference and overload the openQA, the test is extracted from its
# original location and executed on its own dedicated test suites, qam-clamav
# for maintenance and extra_tests_clamav in functional.
~~~